.buildkite/ftr_base_serverless_configs.yml

```diff
@@ -8,11 +8,13 @@ disabled:
   - x-pack/test_serverless/api_integration/config.base.ts
   - x-pack/test_serverless/functional/config.base.ts
   - x-pack/test_serverless/shared/config.base.ts
+  - x-pack/platform/test/serverless/shared/config.base.ts
+  - x-pack/platform/test/serverless/api_integration/config.base.ts
 
 enabled:
   # Serverless deployment-agnostic configs to run platform api-integration tests
   - x-pack/platform/test/api_integration_deployment_agnostic/configs/serverless/oblt.serverless.config.ts
   - x-pack/platform/test/api_integration_deployment_agnostic/configs/serverless/search.serverless.config.ts
   - x-pack/platform/test/api_integration_deployment_agnostic/configs/serverless/security.serverless.config.ts
   - x-pack/platform/test/api_integration_deployment_agnostic/configs/serverless/oblt.logs_essentials.serverless.config.ts
-  - x-pack/platform/test/api_integration_deployment_agnostic/configs/serverless/oblt.streams.serverless.config.ts
\ No newline at end of file
+  - x-pack/platform/test/api_integration_deployment_agnostic/configs/serverless/oblt.streams.serverless.config.ts
```

Comment:
Please check with 8.19: https://github.com/elastic/kibana/blob/8.19/.buildkite/ftr_base_serverless_configs.yml
All serverless must be under `disabled`